exchange/src/exchange/tickers/engine/level_update_generator.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include "exchange/tickers/engine/order_container.hpp"
+#include "shared/messages_exchange_to_wrapper.hpp"
+
+#include <vector>
+
+namespace nutc {
+namespace matching {
+using messages::ObUpdate;
+
+class LevelUpdateGenerator {
```

Comment:
Doesn't have fields
Only method is static

Why is this a class and not a function? This is C++ not Java